docs/dataformats/index.rst

```diff
@@ -19,6 +19,16 @@ we simply use general-purpose file formats define our own semantics as described
 in the :ref:`dataformats_file_formats` section. 
 
 
+.. _time_gammapy:
+
+***************
+Time in Gammapy
+***************
+
+Gammapy adopts the exact same time standard as the Fermi Science Tools as described on this page: `Cicerone: Data - Time in Fermi Data Analysis <http://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Cicerone/Cicerone_Data/Time_in_ScienceTools.html>`_.
```

Comment:
Wrap lines to ~ 80 characters also in RST files.

This section looks out of place, first formats, then time, then you list the formats sub-pages TOC:
https://5c0a1090d7f5da950a4c3a6c72e5dfd9c101960f-www.googledrive.com/host/0BzXHZQx0oCLBfmNwR294aFk5R0NuWlhTaDl5Q2Y0ajFJMGFXX0YxeW1CRlhjQ1FfelhFdEE/dataformats/index.html#time-in-gammapy